C/manual/ch_Finance-Quote.xml

```diff
@@ -332,25 +342,67 @@ Failed to initialize Finance::Quote: missing_modules Finance::Quote JSON::Parse
 </screen>
       </informalexample>
 
-      <para>In this case, &app-fq; is not installed correctly and therefore cannot be used for course retrieval with &app;.
-        Please install &app-fq; according to the instructions at
+      <para>In this case, &app-fq; is not installed correctly and therefore cannot be used for quote retrieval
+        with &app;. Please install &app-fq; according to the instructions at
         <xref linkend="fq-install" />.
       </para>
     </sect2>
 
     <sect2 id="fq-print-quotes">
       <title>Displaying Quotes in a Terminal Window</title>
 
+      <abstract>
+        <para>The <replaceable>--quotes dump</replaceable> option provides quotes for a source and a list of of
+          symbols in a format that is easy for humans to read. This is useful to verify that a
+          particular online quote source is accessible and provides data.
+        </para>
+
+        <para>You can also check to see if the symbol you want to use for your online
+          price retrieval is available at the desired quote source.
+        </para>
+      </abstract>
+
+      <tip>
+        <para>You may use <userinput>gnucash-cli --quotes dump</userinput> to test quote sources and symbols 
+          individually should &app; report an error during quote retrieval. The command will show
+          the fields required by &app;; use <userinput>gnucash-cli --verbose --quotes dump</userinput>
+          to see all of the information returned by &app-fq;.
+        </para>
+      </tip>
+
       <para>To display a quote for one or more stocks or the exchange rate for one or more currencies you can
-        use <userinput>gnucash-cli --quotes dump</userinput> as follows. It offers two output forms
-        for non-currency securities and one for currency exchange rates.
+        use <userinput>gnucash-cli --quotes dump</userinput> as follows:
+        <cmdsynopsis>
+          <command>gnucash-cli</command>
+          <group choice="req">
+            <arg choice="plain">-Q</arg>
+            <arg choice="plain">--quotes</arg>
+          </group>
+          <arg choice="plain">dump</arg>
+          <group choice="opt">
+            <arg choice="plain">-V</arg>
+            <arg choice="plain">--verbose</arg>
+          </group>
+           <arg choice="plain"><replaceable>Source</replaceable></arg>
+           <arg choice="plain" rep="repeat"><replaceable>Symbol</replaceable></arg>
+        </cmdsynopsis>
+      </para>
+
+      <para>It offers two output forms for non-currency securities and one for currency exchange rates.
```

Comment:
Nitpick: did you see
```
gnucash-cli -V --quotes dump  currency USD GBP EUR
Diese Version befindet sich noch in Entwicklung. Sie kann funktionieren, muss aber nicht.
Fehler und andere Probleme werden auf gnucash-devel@gnucash.org diskutiert
Fehlerberichte können hier eingesehen und erstellt werden: https://bugs.gnucash.org
Um die letzte stabile Version zu finden, sehen Sie bitte hier nach: https://www.gnucash.org/
GBP:
      symbol => GBP
    inverted => 0
     success => 1
    currency => USD
        last => 1.28442


EUR:
    inverted => 0
      symbol => EUR
        last => 1.10147
    currency => USD
     success => 1

```

Reply:
@fellen 
You show me the output with the -V switch and i use in this example the short output format.
What did you mean?

Reply:
> What did you mean?

<q>… and **one** for currency exchange rates</q>

Reply:
--verbose doesn't do anything for source currency.

Reply:
Compare the output in https://github.com/Gnucash/gnucash-docs/pull/324#discussion_r1278505507 with that in Currencies of C/gnucash-manual/fq-command-line.html. They are slightly different.